<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeginDialog</class>
 <widget class="QDialog" name="BeginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Resources/img/myGrade.png</normaloff>:/Resources/img/myGrade.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255,255,255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
background-color: transparent;
background: none;
border: none;
background-repeat: none;
color: rgb(255,119,68);
}</string>
   </property>
   <property name="text">
    <string>歡迎</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>271</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>來到我的數獨小遊戲 !!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3
{
background-color: transparent;
background: none;
border: none;
background-repeat: none;
color: rgb(136, 0, 0	);
}</string>
   </property>
   <property name="text">
    <string>助教如果喜歡的話，幫忙順手填個100分喔 !</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>友情提示：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>16</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F
H
P
R
S
U</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_6
{
background-color: transparent;
background: none;
border: none;
background-repeat: none;
color: rgb(0,0,0);
}</string>
   </property>
   <property name="text">
    <string>---------------------&gt;
---------------------&gt;
---------------------&gt;
---------------------&gt;
---------------------&gt;
---------------------&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>81</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_7
{
background-color: transparent;
background: none;
border: none;
background-repeat: none;
color: rgb(0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Finish
Get Hint
Pause
Reset
Solve
Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="begin_cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
